### PR TITLE
Merge some translations from GlitchWitchIO/vulhub-en

### PR DESCRIPTION
**Pull request by GlitchWitchIO**
Merge some translations from GlitchWitchIO/vulhub-en as per vulhub/vulhub#40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Vulhub - Docker-Compose files for vulnerabilities environment
+# Vulhub - Docker-Compose files for creating vulnerable environments
```
[the remark this answers is not in the document]
Comment (phith0n) [reply README.md]: What's the difference between "collection of" and "set of" ?

**Comment by phith0n** [comment]
@GlitchWitchIO 
May I invite you for joining our organization ?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (GlitchWitchIO) [reply README.md]: I think `Vulhub - pre-built vulnerable environments based on Docker-Compose` would be a good change. :+1: 

`collection of` sounds more accurate/professional than `set of` so I am up for changing that too!

However I think the final sentence should look like this 
`Vulhub is an open-source collection of pre-built vulnerable docker environments`

I can push a commit these changes if you both think that's okay.

**Comment by GlitchWitchIO** [comment]
@phith0n Thanks for the positive feedback. You can certainly invite me :smiley:

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (GlitchWitchIO) [reply README.md]: @sfdye 
I pushed the commit feea950 with the changes discussed above. Please let me know what you think.

[the remark this answers is not in the document]
Comment (sfdye) [reply README.md]: Looks good! Now it's up to @phith0n to merge.

**Pull request merged**